### PR TITLE
Update plugin doc

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: development/zh-cn/backend/spi/registry.md [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
+#### 如何扩展
 
-#### FAQ
+`dolphinscheduler-registry-api` 定义了实现插件的标准，当你需要扩展插件的时候只需要实现 `org.apache.dolphinscheduler.registry.api.RegistryFactory` 即可。
 
-1: registry plugin not found
-
-请检查是否有执行 `mvn -U install -Dmaven.test.skip=true` ，此外，请检查配置文件中的 registry.plugin.dir 中配置的目录是否有相关插件。
+`dolphinscheduler-registry-plugin` 模块下是我们目前所提供的注册中心插件。
+#### FAQ
```
Comment (zhongjiajie) [inline development/zh-cn/backend/spi/registry.md:22]: Please add new line before title
```suggestion

#### FAQ
```

**Review by zhongjiajie — COMMENTED** [review]
Most of change look good to me

**Comment by zhongjiajie** [comment]
I would merge to master if no other user comment

File: development/zh-cn/backend/spi/registry.md (hunk 1 above)
Comment (zhongjiajie) [reply development/zh-cn/backend/spi/registry.md:22]: ping @CalvinKirs

Comment (CalvinKirs) [reply development/zh-cn/backend/spi/registry.md:22]: thx

**Comment by zhongjiajie** [comment]
@CalvinKirs Is this PR ready to go now?

**Comment by CalvinKirs** [comment]
> @CalvinKirs Is this PR ready to go now?

yup,PTAL.thx

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**